### DISCUSSION TITLE
Need Help With Biology Thesis Project Involving Gene Annotation Analysis

### DISCUSSION THREAD

**Discussion by jdrswink** [community/community#37739] [Other Feature Feedback, Questions, & Ideas]
Hello all,

My name is Josh Swink and I am a graduate student pursuing a Master of Science in Biology. I am conducting a thesis project on the hypothetical ecological role a novel bacteria might play in it's environment based on it's genome. I have a fully annotated genome from NCBI through the Prokaryotic Genome Annotation Pipeline but I am unsure how to use this data to get a clearer idea of the presence of target genes related to ecological function such as chitinases, nitrogenases, phosphatases, antimicrobial genes, etc. 

Ideally, I would create some sort of pie chart with gene groups based on ecological function and, from those, go into more detail about a protein or two from each group, confirming it's function based on homology with other known proteins, ultimately telling a story about the possible ecological role the bacteria plays based on the presence of these genes. Any advice on how I might proceed would be greatly appreciated!

P.S. - My thesis committee has limited knowledge on whole genome analysis, so that is why I am reaching out here. 

Thanks again for any help you may offer! Take care and I look forward to hearing from any of you. 

Sincerely,
Josh Swink

**Comment by ossanna16** [comment]
Hi @jdrswink,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on your specific request. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
